master/buildbot/scripts/base.py

```diff
@@ -277,6 +278,78 @@ def postOptions(self):
                 msg = 'Required argument missing: ' + missing[0]
             raise usage.UsageError(msg)
 
+    def getUsage(self, width=None):
```

Comment:
I wonder, wouldn't it be possible to use the standard `argparse` module? It has support for subcommands.